README.md

```diff
@@ -5,7 +5,18 @@
 
 [![Release](https://github.com/braintree/braintree-ios-drop-in/actions/workflows/release.yml/badge.svg)](https://github.com/braintree/braintree-ios-drop-in/actions/workflows/release.yml)
 
-Welcome to Braintree's Drop-in SDK for iOS!
+## 📣 Announcements
+🚨🚨🚨 **The iOS Drop-In SDK is currently inactive and will be unsupported by November 2025** 🚨🚨🚨 
```

Comment:
Just a minor suggestion, right now it sounds like it could be unsupported before that, but I think this is a hard date.
```suggestion
🚨🚨🚨 **The iOS Drop-In SDK is currently inactive and will be unsupported on November 1, 2025** 🚨🚨🚨 
```